03/src/components/MenuItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+class MenuItem extends React.Component {
+    render() {
+        return (
+            <li>
+                <a href={this.props.url}>{this.props.text}</a>
```

Comment:
Warto przed `return` użyć destrukturyzacji tj. `const {url, text} = this.props;`